client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -40,6 +51,60 @@ const SitesDataViews = ( { data, isLoading, onViewChange }: SitesDataViewsProps
 
 	const sites = useFormattedSites( data?.sites ?? [] );
 
+	useEffect( () => {
+		if (
+			! window.location.href.includes( 'issue_types=all_issues' ) &&
+			view.filters.some( ( filter ) => filter.value === 1 )
+		) {
+			window.location.href = '/sites?issue_types=all_issues&selectedFilter=1';
+		}
+		if (
+			! window.location.pathname.includes( '/sites/favorites' ) &&
+			view.filters.some( ( filter ) => filter.value === 2 )
+		) {
+			window.location.href = '/sites/favorites?selectedFilter=2';
+		}
+		if (
+			! window.location.href.includes( 'issue_types=backup_failed' ) &&
+			view.filters.some( ( filter ) => filter.value === 3 )
+		) {
+			window.location.href = '/sites?issue_types=backup_failed&selectedFilter=3';
+		}
+		if (
+			! window.location.href.includes( 'issue_types=backup_warning' ) &&
+			view.filters.some( ( filter ) => filter.value === 4 )
+		) {
+			window.location.href = '/sites?issue_types=backup_warning&selectedFilter=4';
+		}
+		if (
+			! window.location.href.includes( 'issue_types=threats_found' ) &&
+			view.filters.some( ( filter ) => filter.value === 5 )
+		) {
+			window.location.href = '/sites?issue_types=threats_found&selectedFilter=5';
+		}
+		if (
+			! window.location.href.includes( 'issue_types=site_disconnected' ) &&
+			view.filters.some( ( filter ) => filter.value === 6 )
+		) {
+			window.location.href = '/sites?issue_types=site_disconnected&selectedFilter=6';
+		}
+		if (
+			! window.location.href.includes( 'issue_types=site_down' ) &&
+			view.filters.some( ( filter ) => filter.value === 7 )
+		) {
+			window.location.href = '/sites?issue_types=site_down&selectedFilter=7';
+		}
+		if (
+			! window.location.href.includes( 'issue_types=plugin_updates' ) &&
+			view.filters.some( ( filter ) => filter.value === 8 )
+		) {
+			window.location.href = '/sites?issue_types=plugin_updates&selectedFilter=8';
+		}
+		if ( ! ( window.location.href === '/sites' ) && view.filters.length === 0 ) {
+			window.location.href = '/sites';
+		}
+	}, [ view.filters ] );
```

Comment:
We should use the `onViewChange` callback from `SitesDashboardV2`, and send there the filter info inside of the `SitesViewState`. Then, `SitesDashboardV2` should do the work, activate the proper filter, do the API call, and change the URL. For reference (copy&paste), `SitesOverview` component is how filter works now, so just copy the same code/behavior to `SitesDashboardV2`.